Xamarin.PropertyEditing/ViewModels/EnumPropertyViewModel.cs

```diff
@@ -11,16 +13,25 @@ internal class EnumPropertyViewModel<TValue>
 		public EnumPropertyViewModel (IPropertyInfo property, IEnumerable<IObjectEditor> editors)
 			: base (property, editors)
 		{
-			PossibleValues = Enum.GetNames (property.Type);
+			var enumNames = Enum.GetNames (property.Type);
 			IsFlags = property.Type.GetCustomAttribute<FlagsAttribute> () != null;
+			if (IsFlags) {
+				var dict = new Dictionary<string, bool> ();
+				foreach (var item in enumNames) {
+					dict.Add (item, (Value as Enum).HasFlag ((Enum)Enum.Parse (property.Type, item)));
+				}
+				PossibleValues = dict;
+			} else {
+				PossibleValues = enumNames.ToDictionary (x => x, y => false);
+			}
 		}
 
 		public bool IsFlags
 		{
 			get;
 		}
 
-		public IReadOnlyList<string> PossibleValues
+		public IReadOnlyDictionary<string, bool> PossibleValues
```

Comment:
This change would break Windows. Whether or not a flag is set doesn't have anything to do with what values are available to set, flags set/unset should be separate.